runtime/src/main/java/org/corfudb/runtime/view/Layout.java

```diff
@@ -107,12 +112,14 @@ public class Layout {
      */
     public Layout(@NonNull List<String> layoutServers, @NonNull List<String> sequencers,
                   @NonNull List<LayoutSegment> segments, @NonNull List<String> unresponsiveServers,
+                  @NonNull List<Pair<Integer,Long>> healProbes,
```

Comment:
nit: Can we have an overloaded constructor instead of modifying the existing constructor to ensure we don't break backward compatibility?